### ISSUE TITLE
Suggestion: Tank Compatibility with Heat Frame

### ISSUE THREAD

**Issue by MuteTiefling** [TeamPneumatic/pnc-repressurized#515]
Just a thought that crossed my mind. Now that you've got tanks with an input/output slot for items, it seems like it could be a nice integration to make them work with heat frame heating and cooling. So slapping a cooled heat frame on a tank full of water, for instance, would start to output blocks of ice.

At the same time, I do enjoy the bit of logistics puzzle I've had to solve in making my ice farm, moving buckets of water around, then shifting the various forms of ice into and out of a pressure chamber. So maybe it's best the way it is. Hard to say. This would greatly simplify the acquisition of the initial ice.

**Comment by desht** [comment]
Actually, the same thought had crossed my mind, but I too am unconvinced about adding that, for pretty much the same reason.

Also, from a technical standpoint, if I were to add cooling of liquids directly, I wouldn't want to make it specific to PNC tanks; it would need to work with any tank.  But then there's the added complexity of ensuring there's somewhere for the output to go - an adjacent inventory maybe, or spawned into the world if there isn't one?

So yeah, not sure about this. I'll keep this open as a reminder, though.

**Comment by MuteTiefling** [comment]
Yeah, I almost deleted the post half way through as I started to think it through. Cross mod incompatibility is another good point against the idea.
